packages/table/test/editableCellTests.tsx

```diff
@@ -26,6 +29,14 @@ describe("<EditableCell>", () => {
         expect(elem.find(".pt-editable-content").text()).to.equal("test-value-5000");
     });
 
+    it("renders loading state", () => {
+        const editableCellHarness = harness.mount(<EditableCell loading={true} value="test-value-5000" />);
+        const editableCellElement = editableCellHarness.element.children[0];
+        expect(editableCellElement.textContent).to.equal("");
+        expect(editableCellElement.classList.contains(Classes.LOADING)).to.be.true;
+        expect(editableCellElement.children[0].classList.contains(Classes.SKELETON)).to.be.true;
```

Comment:
in this case, the skeleton should also be the only child of the cell. i feel like this test should enforce that because the css styles rely on that